dbms/src/Core/Defines.h

```diff
@@ -61,6 +61,7 @@
 #define DBMS_MIN_REVISION_WITH_LOW_CARDINALITY_TYPE 54405
 #define DBMS_MIN_REVISION_WITH_CLIENT_WRITE_INFO 54420
 
+#define DBMS_MIN_REVISION_WITH_CLIENT_SKIPPED_ROWS_INFO 54430
```

Comment:
The current revision is 54429. does that mean we can enable on the next version?

BTW: The test failed because it was not enabled.